PhysicsTools/NanoAOD/python/boostedTaus_cff.py

```diff
@@ -9,9 +9,12 @@
 
 
 finalBoostedTaus = cms.EDFilter("PATTauRefSelector",
-    src = cms.InputTag("slimmedTausBoostedNewID"),
+    src = cms.InputTag("slimmedTausBoosted"),
     cut = cms.string("pt > 40 && tauID('decayModeFindingNewDMs') && (tauID('byVVLooseIsolationMVArun2017v2DBoldDMwLT2017') || tauID('byVVLooseIsolationMVArun2017v2DBoldDMdR0p3wLT2017') || tauID('byVVLooseIsolationMVArun2017v2DBnewDMwLT2017'))")
 )
+run3_nanoAOD_devel.toModify(finalBoostedTaus,
+                            cut = "pt > 40 && tauID('decayModeFindingNewDMs') && (tauID('byVVLooseIsolationMVArun2DBoldDMwLT') || tauID('byVVLooseIsolationMVArun2DBnewDMwLT'))"
+)
```

Comment:
I'm not sure if I understand why the Run3 modifier is needed. Shouldn't the selection on the new IDs be the new default for all NANO?

Reply:
supposing someone was to produce NANO for Run2 again, should the new or old setting be used in this case?